### ISSUE TITLE
NPE in SecurityExtractor when using `null` security scheme

### ISSUE THREAD

**Issue by ddossot** [nidi3/raml-tester#31]
Per [RAML spec](http://raml.org/spec.html#usage-applying-a-security-scheme-to-an-api):

> To indicate that the method may be called without applying any securityScheme, the method may be annotated with the null securityScheme.
> ...
> `securedBy: [null, oauth_2_0]`

When adding `null` to the array of security types, I get:

```
java.lang.NullPointerException
	at guru.nidi.ramltester.core.SecurityExtractor.queryParameters(SecurityExtractor.java:46)
	at guru.nidi.ramltester.core.RamlChecker.checkQueryParameters(RamlChecker.java:146)
	at guru.nidi.ramltester.core.RamlChecker.checkRequest(RamlChecker.java:112)
	at guru.nidi.ramltester.core.RamlChecker.check(RamlChecker.java:68)
```

**Comment by nidi3** [comment]
Your most probably right, I'll have a look at it as soon as I have time (currently at holidays)

**Comment by ddossot** [comment]
Happy to confirm this is now fixed in `0.8.1` :+1:
